src/scripts/frontend-packages-entry.js

```diff
@@ -0,0 +1,16 @@
+// Import GOV.UK Frontend
+import { initAll as initGOVUK } from "govuk-frontend";
+
+// Import MOJ Frontend
+import { initAll as initMOJ } from "@ministryofjustice/frontend";
+
+// Initialize both frontend packages
+if (typeof window !== 'undefined') {
+    console.log('Initializing GOV.UK Frontend...');
```

Comment:
Question: Do you think this might be better as a try catch? That way you get the just the error, rather than the other logs?

```
if (typeof window !== 'undefined') {
    try {
        initGOVUK();
        initMJ();
        console.log('Frontend packages loaded and initialized');
    } catch (error) {
        console.log(error);
    }
}
```